src/content/docs/errors-inbox/apm-tab.mdx

```diff
@@ -0,0 +1,103 @@
+
+The **Errors list view** allows you to:
+
+* Correlate any general patterns on the top 5 errors chart to alerts occurring during the same time period.
+* Dynamic grouping: The default grouping for error occurrences is based on error message, error class, and Transaction UI name. You have the option to change the grouping options by any attribute, up to five attributes at a time.
+* Filtering: Many New Relic customers instrument custom attributes and filtering on a specific custom attribute can be a really quick way to cut through the noise of all error occurrences. 
+
+## Errors details view [#details-view]
+
+Let's say that after you investigated your errors as a whole using the list view, you identified a specific group of hosts that is causing an error spike. You can now use the **details view** to better understand in-context details.
+
+The **Details view** contains: 
+
+* **Applied filters & group-by values:** You may receive a permalink of a specific occurrence. It's important to know the filters and group-by selections to understand the context of the occurrence. 
+* **Profiles:** The profiles tab helps you observe interesting trends across all error occurrences inside a specific error group. You can download this information as a CSV file for further analysis. When you click on an attribute bar, a table is rendered showing the breakdown of attributes and counts.
+
+<img
+  title="error profiles"
+  alt="A depiction of error profiles in the UI"
+  src={apmErrorProfiles}
+/>
+
+### Error occurrences: [#error-occurrences]
+
+The detailed view also allows you to view specific errors. In the top right, you can navigate between the first instance of the error, the last, and any instance in between.
+
+## Triage [#triage-card]
+
+The triage section links the specific error occurrence you're viewing to a system-created error group that has a unique fingerprint. Why does that matter? That unique fingerprint  enables you to triage an error group using a status update or assignment. The system-created error groups are the ones you find on the Triage tab. For more info on how they are generated, see How error groups work.
+
+<img
+  title="triage errors"
+  alt="A depiction of how to tirage errors in APM"
+  src={apmTriageErrors}
+/>
+
+## Select the time period for error data [#selected-time-period]
+
+Use the time picker to examine the details of error events over the past week, month, etc. 
+
+* Error event data in the **Group errors tab** is available for up to a 7 day window of data collected over the last eight days. 
+* Error event metadata in the **Triage tab** is stored up to 13 months. 
+
+You may notice slight differences in count if your time window is set to ending now. This occurs because the counts for the list and table may be requested at slightly different times as the page auto-refreshes.	
+
+## Expected errors [#expected-errors]
+
+
```

Comment:
I like to invert that kind of sentence, so it's more quickly clear why they'd read the sentence. "For more details, see..."